at_onboarding_flutter/example/ios/Runner/Info.plist

```diff
@@ -4,14 +4,16 @@
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
 	<string>$(DEVELOPMENT_LANGUAGE)</string>
```

Comment:
In at_onboarding_flutter/example/ios/Flutter/AppFrameworkInfo.plist this has changed from `$(DEVELOPMENT_LANGUAGE)` to `en`. Should that be consistent here?

Reply:
It shows the default language english in IOS emulator when i tested. I guess this the official flutter project creation code that gives the flexibility to choose any default language for ios side.

Reply:
My question was less about what it does, and more about whether we should consistently use `$(DEVELOPMENT_LANGUAGE)` or `en` as right now some files have one and some the other?

Reply:
All the flutter project creation has similar inconsistency (In AppFrameworkInfo.plist & this file) but we can have a choice to make it consistent.

Reply:
Sounds like a topic for the eng stand (and maybe the architecture call that follows)